AU/Public/Push-Package.ps1

```diff
@@ -10,15 +10,27 @@
     or cached nuget API key.
 #>
 function Push-Package() {
-    $api_key =  if (Test-Path api_key) { gc api_key }
-                elseif (Test-Path ..\api_key) { gc ..\api_key }
-                elseif ($Env:api_key) { $Env:api_key }
-
+    
     $package = ls *.nupkg | sort -Property CreationTime -Descending | select -First 1
     if (!$package) { throw 'There is no nupkg file in the directory'}
-    if ($api_key) {
-        cpush $package.Name --api-key $api_key --source https://push.chocolatey.org 
-    } else {
-        cpush $package.Name --source https://push.chocolatey.org 
+
+    if ($env:au_SimpleServerPath) {
+      try {
+        Move-Item $package.Name $env:au_SimpleServerPath -ea Stop
```

Comment:
A simple server should also use `choco push` and it supports that. This is saying your simple server is on the same machine, which is almost never true.

Reply:
So it would be better to say `au_PushUrl` and default it to `https://push.chocolatey.org/`

Reply:
I am ok, with au_PushUrl for simpleserver usage.   I still think for file share/unc scenarios this is helpful, no?  I have a few scenarios where prior to putting onto a feed we do certification processing, which involves (for better or worse right now) UNC publishing.  We don't have test feeds set up yet.

Reply:
Should be able to choco push to unc paths as well. You may need a fake apikey e.g. `choco push -s \\somewhere\someshare\ -k fake`

Reply:
perhaps the env var needs a better name.  I opted not to create a  Move or Copy-Package routine, but could if that adds more clarity.

Reply:
let me give it a shot.

Reply:
who knew!  awesome.  closing this.